protostar/testing/environments/setup_execution_environment.py

```diff
@@ -10,16 +10,18 @@
 from protostar.testing.starkware.test_execution_state import TestExecutionState
 
 from .common_test_cheatcode_factory import CommonTestCheatcodeFactory
+from ...compiler import ProjectCompiler
```

Comment:
[Style] I think in this case, absolute import would be better. (I use absolute imports when importing from a different module, and relative when importing from the same module.)